easybuild/easyconfigs/a/astro-tulips/astro-tulips-1.0.1-foss-2022a.eb

```diff
@@ -0,0 +1,52 @@
+# Astro-tulips
+
+easyblock = 'PythonBundle'
+
+name = 'astro-tulips'
+version = '1.0.1'
+
+homepage = 'https://astro-tulips.readthedocs.io/en/latest/installation.html'
+description = """tulips creates diagrams of the structure and evolution of stars.
+It creates plots and movies based on output from the MESA stellar evolution code"""
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+toolchainopts = {'pic': True}
```

Comment:
Since this package only seems to have Python code, I don't think this is necessary?
```suggestion
```